SMLHelper/Patchers/CraftDataPatcher.cs

```diff
@@ -88,10 +88,44 @@ internal static void Patch(Harmony harmony)
 #endif
             harmony.Patch(AccessTools.Method(typeof(CraftData), nameof(CraftData.PreparePrefabIDCache)),
                postfix: new HarmonyMethod(AccessTools.Method(typeof(CraftDataPatcher), nameof(CraftDataPrefabIDCachePostfix))));
+            PatchUtils.PatchClass(harmony);
 
             InternalLogger.Log("CraftDataPatcher is done.", LogLevel.Debug);
         }
 
+        [PatchUtils.Prefix]
+        [HarmonyPatch(typeof(CraftData), nameof(CraftData.GetTechType), new Type[] { typeof(GameObject), typeof(GameObject) }, argumentVariations: new ArgumentType[] { ArgumentType.Normal, ArgumentType.Out })]
```

Comment:
This Patch allows the game to load modded techtypes even if the caches all get deleted as it will now get the type from the classid instead of the values of the TechTag